src/ol/control/Attribution.js

```diff
@@ -33,6 +33,8 @@ import {toPromise} from '../functions.js';
  * @property {function(import("../MapEvent.js").default):void} [render] Function called when
  * the control should be re-rendered. This is called in a `requestAnimationFrame`
  * callback.
+ * @property {string|undefined} [staticAttribution] Optional attribution that will always be
```

Comment:
Probably should also allow an array of strings to allow deduplicating multiple attributions.

Reply:
Good point, hopefully that's done now